content/en/blog/_posts/2023-08-15-pkgs-k8s-io-introduction.md

```diff
@@ -173,6 +173,8 @@ publishing packages to the Google-hosted repository in the future.
    curl -fsSL https://pkgs.k8s.io/core:/stable:/v1.28/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg
    ```
 
+   Note: In releases older than Debian 12 and Ubuntu 22.04, the folder `/etc/apt/keyrings` does not exist by default, and it should be created before the curl command.
```

Comment:
nit: for blog posts, we usually like to identify if the section was updated and not in the original blog post
```suggestion
_Update: In releases older than Debian 12 and Ubuntu 22.04, the folder `/etc/apt/keyrings` does not exist by default, and it should be created before the curl command._
```

Reply:
Applied the suggestion, thank you! I can squash commits tomorrow if needed.